lib/server.ml

```diff
@@ -195,6 +195,7 @@ module Command = struct
       ~sw
       ~network
       ~port
+      ?(backlog = 128)
```

Comment:
I'd prefer if we set both the address and the `backlog` in `Server_config.t`. That's actually something I've been meaning to do.

Reply:
Indeed, I added a commit to move the address and backlog to `server_config.t`.